.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -39,16 +39,30 @@ body:
     validations:
       required: true
 
-  - type: textarea
+  - type: dropdown
     id: environment
     attributes:
-      label: What operating system, browser and environment are you using?
-      value: |
-        - **Browser**: Firefox v109
-        - **OS**: Nixos v22.11
-        - **Env**: Local
-    validations:
-      required: true
+      label: On which environments the bug occurs?
+      multiple: true
+      options:
+        - Local
+        - Dev (DgexSol)
+        - Staging (DgexSol)
+        - Dev (SNCF)
+        - Recette (SNCF)
+        - Integration (SNCF)
+
+  - type: dropdown
+    id: Browser
+    attributes:
+      label: On which browser the bug occurs?
+      multiple: true
+      options:
+        - Firefox
+        - Google Chrome
+        - Microsoft Edge
+        - Safari
+        - Opera
```

Comment:
Maybe we could have an "Other" option here, just because there will always be That Browser That Isn't In The Big List Of All Browsers?

Reply:
Done